presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleConnector.java

```diff
@@ -137,12 +145,27 @@ public ConnectorPageSinkProvider getPageSinkProvider()
                         value -> ImmutableList.copyOf(((List<String>) value).stream()
                                 .map(name -> name.toLowerCase(ENGLISH))
                                 .collect(toList())),
-                        List.class::cast));
+                        List.class::cast),
+                new PropertyMetadata<>(
+                        PAGE_PROCESSING_DELAY,
+                        "Sleep duration before processing each page",
+                        VARCHAR,
+                        Duration.class,
+                        Duration.valueOf("0s"),
```

Comment:
It's better not to use the string constructor for constants `new Duration(0, SECONDS)`